include/flexflow/ops/element_unary_params.h

```diff
@@ -0,0 +1,28 @@
+#ifndef _FLEXFLOW_ELEMENTARY_UNARY_PARAMS_H
+#define _FLEXFLOW_ELEMENTARY_UNARY_PARAMS_H
+
+#include "flexflow/ffconst.h"
+#include "flexflow/parallel_tensor.h"
+
+namespace FlexFlow {
+
+struct ElementUnaryParams {
+  OperatorType op_type;
+  bool inplace;
+  float scalar;
```

Comment:
I don't think all unary ops have a `scalar`--is there a change this will create problems?

Reply:
I'll add a default value. I believe this is how it's currently handled in the other constructor based on the implementation of `FFModel::unary`